mobile/ios/ios-device.ts

```diff
@@ -295,7 +297,7 @@ export class IOSDevice implements Mobile.IIOSDevice {
 				};
 
 				try {
-			    	this.tryToExecuteFunction<void>(func);
+					this.tryToExecuteFunction<void>(func);
```

Comment:
I found a bug in unrelated code part and committed a fix in https://github.com/telerik/mobile-cli-lib/pull/201 If my change is merged before your PR, you'll need to rebase.